### ISSUE TITLE
debugEvents missing, causing error

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#3088]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.62.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.62.1...4.62.2) (2023-03-30)

- Release 4.62.2 [`1ae330d`](https://github.com/open-wa/wa-automate-nodejs/commit/1ae330d35b61d4f766e73cf41ebf0540d2b649cf)
- 🚑 fix: #3088 [`10c18c3`](https://github.com/open-wa/wa-automate-nodejs/commit/10c18c3cd6200df8d28f20bae77505d112fa2904)
- updated types-only package [`35dde20`](https://github.com/open-wa/wa-automate-nodejs/commit/35dde2023c68620553fb844bfa80d2df89652f62)
